test/integration/data_bags/osl_jenkins/powerci.json

```diff
@@ -1,38 +0,0 @@
-{
-  "id": "powerci",
-  "admin_users": {
```

Comment:
Was there a reason you deleted this file? I am hesitant to reinstate it in case the recipe does something that is `not good`. I say this because my kitchen tests fail on indexing the credentials array.

Reply:
This was an improper way of testing this using test-kitchen and should be using the environment variables instead as stated in the kitchen.yml. I also added two new vars which are for Github oauth creds. We have this mostly documented in the kitchen.yml however given this feedback perhaps I should move that to the README and also update it.